Userland/Applications/Calculator/Calculator.cpp

```diff
@@ -34,23 +34,23 @@ KeypadValue Calculator::begin_operation(Operation operation, KeypadValue argumen
         return argument;
 
     case Operation::Sqrt:
-        if (argument < KeypadValue {}) {
+        if (argument < Crypto::BigFraction {}) {
             m_has_error = true;
             return argument;
         }
-        res = argument.sqrt();
+        res = Crypto::BigFraction { AK::sqrt(argument.to_double()) };
```

Comment:
Can we make this some member function such as `Crypto::BigFraction::inexact_sqrt()`?